samples/net/gptp/README.rst

```diff
@@ -33,7 +33,7 @@ Follow these steps to build the gPTP sample application:
 .. zephyr-app-commands::
    :zephyr-app: samples/net/gptp
    :board: <board to use>
-   :conf: prj.conf
+   :conf: prj_base.conf
```

Comment:
Why are you changing this?

Reply:
Good catch, but that should be placed in separate commit as it is unrelated to mimix1050rt support.